core/unit_test/TestGraph.hpp

```diff
@@ -219,6 +216,20 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph),
   ASSERT_EQ(0, bugs_host());
 }
 
+// This test ensures that it's possible to build a Kokkos::Graph using
+// Kokkos::Experimental::create_graph without providing a closure.
+TEST_F(TEST_CATEGORY_FIXTURE(graph), create_graph_no_closure) {
+  auto graph = Kokkos::Experimental::create_graph<TEST_EXECSPACE>(ex);
```

Comment:
Make sure you have one with an exec argument, and another assertion w/o it.

Reply:
:heavy_check_mark: